stremio.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Name=Stremio
+Comment=Video organizer for your Movies, TV Shows and TV Channels
+Exec=./stremio %U
+Icon=stremio
+Terminal=false
+Type=Application
+Categories=GNOME;GTK;AudioVideo;
```

Comment:
The categories are incorrect. The application doesn't use Gtk nor it's an application that follows GNOME's HIG.

Reply:
thanks for noticing this, it might've been an artifact from Electron times, but even then "Gnome" would've been wrong

Reply:
@Ivshti Is there a way to contact devs about another packaging platform or to help make the Linux users' experience much better? 
Thanks!

Reply:
just open an issue in stremio-shell